src/plugins/controls/public/time_slider/embeddable/time_slider_embeddable.tsx

```diff
@@ -50,6 +50,12 @@ export class TimeSliderControlEmbeddable extends Embeddable<
   private getTimezone: ControlsSettingsService['getTimezone'];
   private timefilter: ControlsDataService['timefilter'];
   private prevTimeRange: TimeRange | undefined;
+  private prevTimesliceAsPercentage:
```

Comment:
prevTimesliceAsPercentage needs to be set either in constructor or syncWithTimeRange (which is called by constructor)

Reply:
Good catch! Done in 3297a73aed91ae4c96fb9da1db8e2b779e2f0870 👍

Reply:
Right WOOP sorry :) Done in 3297a73aed91ae4c96fb9da1db8e2b779e2f0870